### PR TITLE
if preview declined the url should not dissapear

### PR DESCRIPTION
**Pull request by asadazam93**
### Description
On the course run detail page if the course team declines the preview, the URL does not disappear, but that field is able to be edited by publisher

**Sandbox**
https://discovery-escalation-discovery.sandbox.edx.org/

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] @schenedx    
- [ ] @awaisdar001

### Post-review
- [ ] Rebase and squash commits

### CODE REVIEW

File: course_discovery/apps/publisher/templates/publisher/_approval_widget.html [hunk 1]
```diff
@@ -20,7 +20,7 @@ <h5 class="hd-5 emphasized approvals-heading">{% trans "REVIEWS" %}</h5>
                 <div class="layout-1q3q layout-reversed">
                     <div class="layout-col layout-col-a">
                     {% if object.preview_url %}
-                        {% if object.preview_url and object.course.course_team_admin == request.user and object.course_run_state.is_approved and not object.course_run_state.is_preview_accepted %}
+                        {% if object.preview_url and object.course.course_team_admin == request.user and object.course_run_state.is_approved and not object.course_run_state.is_preview_accepted and object.course_run_state.owner_role == 'course_team' %}
```
Comment (awaisdar001) [inline course_discovery/apps/publisher/templates/publisher/_approval_widget.html]: we are checking `object.preview_url` on line#22 do we need to check it again here?

**Review by awaisdar001 — CHANGES_REQUESTED** [review]

File: course_discovery/apps/publisher/templates/publisher/_approval_widget.html (hunk 1 above)
Comment (awaisdar001) [reply course_discovery/apps/publisher/templates/publisher/_approval_widget.html]: what is `object` here?

File: course_discovery/apps/publisher_comments/models.py [hunk 2]
```diff
@@ -47,10 +47,6 @@ def save(self, *args, **kwargs):
 @transaction.atomic
 def mark_preview_url_as_decline(instance):
     course_run = instance.content_type.get_object_for_this_type(pk=instance.object_pk)
-
-    # remove the preview url
-    preview_url = course_run.preview_url
-    course_run.preview_url = None
```
Comment (schenedx) [inline course_discovery/apps/publisher_comments/models.py:53]: After the removal of these lines, we make this function useless. Why have this function?

**Review by schenedx — CHANGES_REQUESTED** [review]
I think the change can be designed better

File: course_discovery/apps/publisher_comments/models.py (hunk 2 above)
Comment (asadazam93) [reply course_discovery/apps/publisher_comments/models.py:53]: We still have to change owner_role and send email in this function.

File: course_discovery/apps/publisher/wrappers.py [hunk 3]
```diff
@@ -142,6 +142,12 @@ def course_type(self):
             return Seat.PROFESSIONAL
         return Seat.AUDIT
 
+    @property
+    def is_in_preview_review(self):
```
Comment (awaisdar001) [inline course_discovery/apps/publisher/wrappers.py:146]: @asadazam93 
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines
pls follow for breaking long lines.

**Comment by awaisdar001** [comment]
Just a nit, else looks good to me, Do you have an updated sandbox link?

**Comment by asadazam93** [comment]
@awaisdar001 Sandbox updated

**Review by awaisdar001 — APPROVED** [review]

File: course_discovery/apps/publisher_comments/models.py (hunk 2 above)
Comment (schenedx) [reply course_discovery/apps/publisher_comments/models.py:53]: @asadazam93 Got it. Then we don't need to save the course_run on line 50

**Review by schenedx — APPROVED** [review]
Just 1 more comment. Rest LGTM

**Pull request merged**